google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -1215,7 +1218,8 @@ def iter_crm_organization_org_policies(self, org_id):
             '//cloudresourcemanager.googleapis.com/{}'.format(org_id),
             self.engine)
         for org_policy in resources:
-            yield org_policy
+            data, metadata = org_policy
+            yield data[0], metadata
```

Comment:
please put in comment as why [0] is needed, same for the other 2 methods.

Reply:
Done